README.md

```diff
@@ -1,28 +1,28 @@
 # stoa
 API server for Agora
 
+# Build instructions
+Install the Node.js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
```

Comment:
```suggestion
Install Node.js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
```
Since `Node.js >= 12.18.0` is defined, it doesn't need an article.
You only need an article when the name is not defined, e.g. it is "a category".
You say "install the game" or "install a game", but also "install Lineage", because "Lineage" is "the game".

Reply:
Additionally, `>=` means that one might want to use a newer version ;)

Reply:
The corresponding version statement has been deleted to use the latest version.